packages/labs/react/web-test-runner.config.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
File level comment/question: So React is now tested separately, hence the removal from the `packages/test` directory?

Reply:
Correct. It needs a separate WTR config than the others in `packages/tests` so is run separately.